tests/fixtures/noqa/noqa.py

```diff
@@ -788,3 +788,20 @@ def get_item():  # noqa: WPS463
    some(number) for numbers in matrix
    for number in numbers # noqa: WPS361
 ]
+
+def detect_bare_raise1():
+    """Function to check if the bare raise is detected."""
+    raise
+
+def detect_bare_raise2_helper():
+    """Function to check if the bare raise is detected."""
+    raise ZeroDivisionError
+
+def detect_bare_raise2():
+    """Function to check if the bare raise is detected."""
+    try:
+        testvar = 1/0  # noqa: WPS344, F841
+    except ZeroDivisionError:
+        detect_bare_raise2()
+        raise ZeroDivisionError
+
```

Comment:
Let's leave here just one function with bare `raise` like mentioned in https://github.com/wemake-services/wemake-python-styleguide/pull/1998/files#r619652218